prompts.py

```diff
@@ -0,0 +1,59 @@
+orchestrator_prompt = """
+You are an AI assistant made for information analysis.\n
+Let's think step by step.\n
+Step 1: Look at the image provided.\n
+Step 2: Think about what is a pie chart, what is a bar chart, what is a line graph, what is a list before you respond.\n
+Step 3: Identify the type of chart in the image.\n
+Step 4: Check if what you think is correct.\n
+Step 5: Respond with "pie chart" if it is a pie chart, "bar chart" if it is a bar chart, "line graph" if it is a line graph, "list" if it is a list, "trash" if it is anything else.\n
+"""
+
+pie_chart_prompt =  """
+You are working at a firm and you will be given a pie chart. 
+Since this pie chart is very important, you need to focus on it.
+
+1. First, find out how many different parts this pie chart contains. This should be a number.
+2. Then, verify that the number you got is correct.
+3. Next, find out the values, labels, and colors for each part.
+4. Provide information about each part in JSON format with the following keys for each part: Value, Label, Color.
+5. Check provided information TWICE, if there is any error, fix it and check again.
+6. Put ! at the beggining and at the end of JSON format.
+
+This is an example of correctly formatted JSON output:
+!{
+"Label1": {
+"Value": val1, 
+"Label": lab1, 
+"Color": col1
+},
+"Label2": {
+"Value": val2, 
+"Label": lab2, 
+"Color": col2
+},
+"Label3": {
+"Value": val3, 
+"Label": lab3, 
+"Color": col3
+},
+...,
+"LabelN": {
+"Value": valN, 
+"Label": labN, 
+"Color": colN
+}
+}!
+
+This example is not related to your case; it is just an example of the format.
+
+Be careful when you read values and labels, and double-check to ensure the data you found is accurate.
```

Comment:
Ove stvari koje su mu neka upuststva i pravila napisati u formatu:

While generating the response, follow these rules:
1)
2)
...